tools/build/build.sh

```diff
@@ -45,8 +45,8 @@ build_ubuntu_kernel() {
     else
         CHANGELOG="debian.master/changelog"
     fi
-    LATEST_VERSION=$(sed -n '1 s/linux.*(\(.*\)) noble.*$/\1/p' ${CHANGELOG})
-    CCNP_VERSION=${LATEST_VERSION}${CCNP_VERSION_SUFFIX}
+    LATEST_VERSION=$(sed -n '1 s/\(linux.*(.*\)) noble.*$/\1/p' ${CHANGELOG})
+    CCNP_VERSION="${LATEST_VERSION}${CCNP_VERSION_SUFFIX}\)"
```

Comment:
why need this blanket?

Reply:
> It's compatible with two kernel versions:
> 
> * for generic kernel, the default version in changelog is `linux (6.8.0-31.31)`, we want to change to ccnp version `linux (6.8.0-31.31+ccnp1)`
> * for intel kernel, the default version in changelog is `linux-intel (6.8.0-1001.7)`, we want to change it to `linux-intel (6.8.0-1001.7+ccnp1)`
>   that's why we use `sed` to get the version `linux (6.8.0-31.31` or `linux-intel (6.8.0-1001.7`, then add the suffix with right one.

So the last bracket is required for the concatenation of $LATEST_VERSION and fixed $CCNP_VERSION_SUFFIX?

Reply:
Yes.